apisix/plugins/api-aggregate.lua

```diff
@@ -0,0 +1,144 @@
+--
+-- Licensed to the Apache Software Foundation (ASF) under one or more
+-- contributor license agreements.  See the NOTICE file distributed with
+-- this work for additional information regarding copyright ownership.
+-- The ASF licenses this file to You under the Apache License, Version 2.0
+-- (the "License"); you may not use this file except in compliance with
+-- the License.  You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+local core   = require("apisix.core")
+local http   = require("resty.http")
+local ngx    = ngx
+local ipairs = ipairs
+local pairs  = pairs
+local type   = type
+
+local plugin_name = "api-aggregate"
+
+local schema = {
+    type = "object",
+    properties = {
```

Comment:
we need a schema

Reply:
@justinmclean please use a valid schema

example: https://github.com/apache/incubator-apisix/blob/master/apisix/schema_def.lua

Reply:
@membphis Hi, What should I do for fixing it?

Reply:
sorry for this. the current implementation is right.